config/200-config.yaml

```
@@ -60,7 +60,7 @@ data:
     cluster-cert-secret: ""
 
     # Specifies the amount of time that Kourier waits for the incoming requests.
-    stream-idle-timeout: "300s"
+    stream-idle-timeout: "infinity"
```

```suggestion
    # The default, 0s, imposes no timeout at all.
    stream-idle-timeout: "0s"
```

I think we can just put a comment about `0s` without `asKourierDuration()` which is simple, isn't it?